app/controllers/questions_controller.rb

```diff
@@ -10,28 +10,63 @@ def index
     @question_limitations = ENV["limit_question_index_to_users_questions_only"]
   end
 
+  def show
+    @questions = get_question_list()
+    @question_id = params[:id]
+    @question = @questions[(params[:id]).to_i - 1]["text"]
+
+    logger.debug "QUESTION's type in question's show: #{@questions[(params[:id]).to_i - 1].class}"
```

Comment:
This can use `@question` directly, since you've already made a reference to it in line 16 -- no need for the much longer `@questions[(params[:id]).to_i - 1]`, which another developer has to look at closely to verify is actually the same thing as `@question`.